test/json/component_status_list.json

```diff
@@ -0,0 +1,52 @@
+{
+  "kind": "ComponentStatusList",
+  "apiVersion": "v1",
+  "metadata": {
+    "selfLink": "/api/v1/componentstatuses"
+  },
+  "items": [
+    {
+      "metadata": {
+        "name": "controller-manager",
+        "selfLink": "/api/v1/namespaces/componentstatuses/controller-manager",
+        "creationTimestamp": null
+      },
+      "conditions": [
+        {
+          "type": "Healthy",
+          "status": "Unknown",
+          "error": "Get http://127.0.0.1:10252/healthz: dial tcp 127.0.0.1:10252: connection refused"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "name": "scheduler",
+        "selfLink": "/api/v1/namespaces/componentstatuses/scheduler",
+        "creationTimestamp": null
+      },
+      "conditions": [
+        {
+          "type": "Healthy",
+          "status": "Unknown",
+          "error": "Get http://127.0.0.1:10251/healthz: dial tcp 127.0.0.1:10251: connection refused"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "name": "etcd-0",
+        "selfLink": "/api/v1/namespaces/componentstatuses/etcd-0",
```

Comment:
this url suggests that component status is actually a name of a special namespace, rather than a resource of type component status on its own. 
is that the case?